modules/bridge/common/bridge_buffer_test.cc

```diff
@@ -0,0 +1,57 @@
+/******************************************************************************
+ * Copyright 2019 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+
+#include <stdio.h>
+#include <gtest/gtest.h>
+#include <string>
+#include "modules/bridge/common/bridge_buffer.h"
+
+namespace apollo {
+namespace bridge {
+
+TEST(BridgeBufferTest, bridge_buf_test) {
+  BridgeBuffer<char> buf;
+  char *p = buf;
+  EXPECT_EQ(0, buf.capacity());
+  EXPECT_EQ(0, buf.size());
+  EXPECT_TRUE((buf == nullptr));
```

Comment:
1. You don't need the inner `==`.
2. Use `EXPECT_EQ`, `EXPECT_NE` for such cases.